### PR TITLE
Feature/issue 6473/update to php 8 3

### PR DESCRIPTION
**Pull request by Nyanko-sensei**
resolves #6473

### Changelist

- [ ] Added loosen up constrains for mailersend/mailersend

### Testing/QA

- Not testable

### How to release

Nothing special

### PR Checklist

- [ ] Tests are done and pass
- [ ] No PHP and JS errors
- [ ] PR does not go out of scope of the task
- [ ] Possible breaking changes are documented in the PR

Add other relevant [Pull Request guidelines](https://github.com/mailerlite/mailersend/blob/master/docs/project/PULL-REQUEST-GUIDELINES.md) checkpoints.

### Screenshots

None

### Risks

Not testable, so it may break 

### Additional notes

None

### CODE REVIEW

File: tests/MailerSendTransportTest.php [hunk 1]
```diff
@@ -111,6 +111,10 @@ public function test_get_recipients(): void
 
     public function test_get_attachments(): void
     {
+        if (!method_exists(new \Symfony\Component\Mime\Email(), 'attachPart')) {
+            $this->markTestSkipped('Skipped as test uses depricated method');
```
Comment (fosron) [inline tests/MailerSendTransportTest.php:115]:
```suggestion
            $this->markTestSkipped('Skipped as test uses deprecated method');
```

File: tests/MailerSendTransportTest.php [hunk 2]
```diff
@@ -130,6 +134,10 @@ public function test_get_attachments(): void
 
     public function test_get_additional_data(): void
     {
+        if (!method_exists(new \Symfony\Component\Mime\Email(), 'attachPart')) {
+            $this->markTestSkipped('Skipped as test uses depricated method');
```
Comment (fosron) [inline tests/MailerSendTransportTest.php:138]:
```suggestion
            $this->markTestSkipped('Skipped as test uses deprecated method');
```

File: tests/MailerSendTransportTest.php (hunk 1 above)
Comment (Nyanko-sensei) [reply tests/MailerSendTransportTest.php]: edited to not skip this test

File: tests/MailerSendTransportTest.php (hunk 2 above)
Comment (Nyanko-sensei) [reply tests/MailerSendTransportTest.php]: edited to not skip this test

**Review by fosron — APPROVED** [review]

**Pull request merged**